tests/sympc/approximations/log_test.py

```diff
@@ -0,0 +1,33 @@
+# third party
+import torch
+
+from sympc.approximations.log import log
+from sympc.config import Config
+from sympc.session import Session
+from sympc.session import SessionManager
+from sympc.tensor.mpc_tensor import MPCTensor
+
+
+def test_log(get_clients) -> None:
+    clients = get_clients(2)
+    session_one = Session(parties=clients)
+    SessionManager.setup_mpc(session_one)
+
+    x_secret = torch.Tensor([0.1, 0.5, 2, 5, 10, 20, 50, 100, 250])
+
+    # default precision
+    x = MPCTensor(secret=x_secret, session=session_one)
+    x_log = log(x)
+    x_secret_log = torch.log(x_secret)
+
+    assert torch.allclose(x_secret_log, x_log.reconstruct(), rtol=0.4)
```

Comment:
rtol should be somewhere as ```1e-3``` or ```1e-4```. 0.4 is a big value

Reply:
For all the cases

Reply:
updated